content/community.json

```diff
@@ -52,7 +55,7 @@
       "pictureUrl": "/img/community/ugc/medium_logo.png"
     },
     {
-      "url": "https://blog.codecentric.de/en/2020/01/remote-training-gitlab-ci-dvc/",
+      "url": "https://mribeirodantas.xyz/blog/index.php/2020/03/05/r-dvc-and-rmarkdown/",
```

Comment:
let's also update the `Women in Data Science San Diego` date - let's set Friday, March 27, 2020 for now . I'm not sure if we'll attend this event at all now

Reply:
Do we want to do this here, or in another smaller PR that may get in quicker? I'm fine doing it either way.

Reply:
here is fine

Reply:
Done !